metadata.json

```diff
@@ -69,6 +75,7 @@
     }
   ],
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":">=3.2.0 <5.0.0"}
+    {"name":"puppetlabs/stdlib","version_requirement":">=3.2.0 <5.0.0"},
+    {"name":"zleslie/pkgng","version_requirement":">=0.2.5 <2.0.0"}
```

Comment:
-1 to requiring zleslie/pkgng only because it is still < 1.0.0

Could you use the supported freebsd or ports providers?